src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -901,4 +902,144 @@ final case class EServicesApiServiceImpl(
+        ),
+        "riskAnalysis" -> latestRiskAnalysis
+          .map(ra =>
+            Json.obj(
+              "version"       -> Json.fromString(ra.version),
+              "singleAnswers" -> Json.arr(
+                ra.singleAnswers.map(sa =>
+                  Json.obj(
+                    "key"   -> Json.fromString(sa.key),
+                    "value" -> sa.value.map(Json.fromString).getOrElse(Json.obj())
+                  )
+                ): _*
+              ),
+              "multiAnswers"  -> Json.arr(
+                ra.multiAnswers.map(ma =>
+                  Json.obj("key" -> Json.fromString(ma.key), "values" -> Json.arr(ma.values.map(Json.fromString): _*))
+                ): _*
+              )
+            )
+          )
+          .getOrElse(Json.obj())
+      ).asJson
+    }
+
+    def checkRiskAnalysisForExport(eService: CatalogProcess.EService): Future[Unit] =
+      eService.mode match {
+        case CatalogProcess.EServiceMode.RECEIVE if eService.riskAnalysis.isEmpty =>
+          Future.failed(EServiceRiskAnalysisIsRequired(eService.id))
+        case _                                                                    => Future.unit
+      }
+
+    def verifyExportEligibility(descriptor: CatalogProcess.EServiceDescriptor): Future[Unit] =
+      descriptor.state match {
+        case CatalogProcess.EServiceDescriptorState.DRAFT =>
+          Future.failed(NotValidDescriptor(descriptor.id.toString, descriptor.state.toString))
+        case _                                            => Future.unit
+      }
+
+    def createZip(
+      folderName: String,
+      docs: Seq[(Array[Byte], String)],
+      interfaceFile: (Array[Byte], String),
+      config: Json
+    ): Array[Byte] = {
+      val bos    = new ByteArrayOutputStream()
+      val zipOut = new ZipOutputStream(bos)
+
+      docs.foreach { case (data, entryName) =>
+        zipOut.putNextEntry(new ZipEntry(s"$folderName/documents/$entryName"))
+        zipOut.write(data)
+        zipOut.closeEntry()
+      }
+
+      val (interfaceData, interfaceFileName) = interfaceFile
+      zipOut.putNextEntry(new ZipEntry(s"$folderName/$interfaceFileName"))
+      zipOut.write(interfaceData)
+      zipOut.closeEntry()
+
+      zipOut.putNextEntry(new ZipEntry(s"$folderName/configuration.json"))
+      zipOut.write(config.toString.getBytes("UTF-8"))
+      zipOut.closeEntry()
+
+      zipOut.close()
+      bos.toByteArray
+    }
+
+    val result: Future[(String, HttpEntity.Strict)] = for {
+      organizationId <- getOrganizationIdFutureUUID(contexts)
+      eServiceUuid   <- eserviceId.toFutureUUID
+      descriptorUuid <- descriptorId.toFutureUUID
+      eService       <- catalogProcessService.getEServiceById(eServiceUuid)
+      _              <- assertRequesterAllowed(eService.producerId)(organizationId)
+      _              <- checkRiskAnalysisForExport(eService)
```

Comment:
just curious, which is the purpose of performing the risk analysis check during the export?
it's not wrong of course, do we need to prevent a specific case?
Some considerations:
- the check is duplicated and in case of changes it should be updated here as well, potentially introducing bugs
- even if the eservice was published in mode Receive without risk analysis, once imported it would be in Draft state, which allows not having the risk analysis yet, so it does not create inconsistent states

Reply:
> just curious, which is the purpose of performing the risk analysis check during the export? it's not wrong of course, do we need to prevent a specific case? Some considerations:
> 
>     * the check is duplicated and in case of changes it should be updated here as well, potentially introducing bugs
> 
>     * even if the eservice was published in mode Receive without risk analysis, once imported it would be in Draft state, which allows not having the risk analysis yet, so it does not create inconsistent states

I agree, the only check we should mantain on the export is to not allow it on DRAFTs. 
For the rest we should trust the state on the platform.

If @beetlecrunch agrees too we should remove these check from the ticket too in order to be coherent with the implementation.